addons/l10n_br/models/res_company.py

```diff
@@ -12,7 +12,12 @@ class ResCompany(models.Model):
     l10n_br_ie_code = fields.Char(string="IE", help="State Tax Identification Number. Should contain 9-14 digits.") # each state has its own format. Not all of the validation rules can be easily found.
     l10n_br_im_code = fields.Char(string="IM", help="Municipal Tax Identification Number") # each municipality has its own format. There is no information about validation anywhere.
     l10n_br_nire_code = fields.Char(string="NIRE", help="State Commercial Identification Number. Should contain 11 digits.")
+    show_taxes = fields.Boolean(compute='_set_show_taxes', store=False, default=True)
```

Comment:
compute method should start with _compute_..., also do we really need the default with the compute ?